### PR TITLE
fix: Moved Search bar from patterns to components

### PR DESCRIPTION
**Pull request by jlongshore**
Alphabetized the components/patterns in various sections

#### What did you change?
story-structure.js <-- modified the order to be alphabetical. 

#### How did you test and verify your work?
Storybook

### CODE REVIEW

File: packages/core/story-structure.js [hunk 1]
```diff
@@ -53,29 +53,34 @@ const s = [
...
-          { n: 'Tearsheet', s: ['c/Tearsheet', 'c/TearsheetNarrow'] },
-          { n: 'User avatar', s: ['c/UserAvatar'] },
           {
             n: 'StatusIndicator',
             s: ['c/StatusIndicator', 'c/StatusIndicatorStep'],
           },
+          { n: 'String formatter', s: ['c/StringFormatter'] },
+          { n: 'Tag set', s: ['c/TagSet'] },
+          { n: 'Tearsheet', s: ['c/Tearsheet', 'c/TearsheetNarrow'] },
+          { n: 'TruncatedList', s: ['c/TruncatedList'] },
```
Comment (matthewgallo) [inline packages/core/story-structure.js:74]:
```suggestion
          { n: 'Truncated list', s: ['c/TruncatedList'] },
```

File: packages/core/story-structure.js [hunk 2]
```diff
@@ -53,29 +53,34 @@ const s = [
             ],
           },
           { n: 'DataSpreadsheet', s: ['c/DataSpreadsheet'] },
-          { n: 'TruncatedList', s: ['c/TruncatedList'] },
-          { n: 'DelimitedList', s: ['c/DelimitedList'] },
           { n: 'Decorator', s: ['c/Decorator'] },
+          { n: 'DelimitedList', s: ['c/DelimitedList'] },
```
Comment (matthewgallo) [inline packages/core/story-structure.js:57]:
```suggestion
          { n: 'Delimited list', s: ['c/DelimitedList'] },
```

**Review by matthewgallo — CHANGES_REQUESTED** [review]
Just spotted two small sentence case changes 👀

File: packages/core/story-structure.js (hunk 2 above)
Comment (jlongshore) [reply packages/core/story-structure.js]: Good catch, I found a few others ( Inlinetip and Non-linear reading )

**Review by matthewgallo — APPROVED** [review]

**Pull request merged**